packages/perseus/src/widgets/radio/multiple-choice-component.new.tsx

```diff
@@ -72,208 +37,102 @@ interface MultipleChoiceComponentProps {
  * Created as part of the Radio Revitalization Project (LEMS-2933).
  */
 const MultipleChoiceComponent = ({
-    apiOptions,
-    reviewModeRubric,
-    reviewMode,
-    editMode = false,
-    multipleSelect = false,
-    labelWrap,
+    choices,
     countChoices,
+    multipleSelect = false,
     numCorrect,
-    isLastUsedWidget,
-    choices,
     onChoiceChange,
+    reviewMode,
 }: MultipleChoiceComponentProps): React.ReactElement => {
     const {strings} = usePerseusI18n();
 
-    const choiceRefs = useRef<Array<React.RefObject<HTMLButtonElement>>>([]);
-    // Keep track of the previous review mode rubric to avoid unnecessary
-    // scrolling when switching between review mode and edit mode.
-    const prevReviewModeRubric = useRef<
-        PerseusRadioWidgetOptions | undefined | null
-    >();
-
-    useEffect(() => {
-        // Switching into review mode can sometimes cause the selected answer
-        // to scroll out of view - for example, when we reveal all those tall
-        // rationales. This can be disorienting for the user.
-        //
-        // So, if this widget just switched into review mode, ensure that the
-        // checked choice (or first checked choice, if there are multiple) is
-        // scrolled into view.
-        //
-        // This only happens if the `canScrollPage` API option is set (so that
-        // call sites aren't surprised by scrolling), and if this widget was
-        // the most recently used widget (because, if the user has since
-        // touched something else, they're probably not trying to keep their
-        // eye on this widget anymore).
-        if (
-            apiOptions.canScrollPage &&
-            isLastUsedWidget &&
-            reviewModeRubric &&
-            !prevReviewModeRubric.current
-        ) {
-            const checkedIndex = choices.findIndex((c) => c.checked);
-            if (checkedIndex >= 0) {
-                const ref = choiceRefs.current[checkedIndex];
-
-                if (ref.current) {
-                    scrollElementIntoView(ref.current);
-                }
-            }
-        }
-
-        prevReviewModeRubric.current = reviewModeRubric;
-    }, [apiOptions, isLastUsedWidget, reviewModeRubric, choices]);
-
-    const choiceListClassName = classNames(
-        "perseus-widget-radio",
-        // TODO(LEMS-3229): Remove this line after we stop using BaseRadio in editor
-        !editMode && "perseus-rendered-radio", // Styles to be applied when not in the editor
-        styles.choiceList, // Main class for the choice list
-    );
-
-    const instructionsClassName = classNames(styles.instructions);
     const instructions = getInstructionsText({
         multipleSelect,
         countChoices,
         numCorrect,
         strings,
     });
 
-    const fieldset = (
+    const choiceListClasses = [styles.choiceList]
+        .concat(reviewMode ? [styles.reviewAnswers] : [])
+        .join(" ");
```

Comment:
I'm finding this a little hard to parse? Could this just be
```suggestion
    const choiceListClasses = reviewMode
        ? `${styles.choiceList} ${styles.reviewAnswers}`
        : styles.choiceList;
```
?